src/persistence/basin_subzone.js

```diff
@@ -202,5 +202,45 @@ module.exports = (
       )
         .then(layers => layers.rows[0].collection)
     ),
+
+    /**
+     * Get the persistence human footprint layer divided by categories in a given basin subzone
+     * @param {Number} subzoneId basin subzone id
+     *
+     * @return {Object} Geojson object with the geometry
+     */
+    findHFPersistenceLayerById: subzoneId => (
+      db.raw(
+        `SELECT row_to_json(fc) AS collection
+        FROM (
+          SELECT 'FeatureCollection' AS type, array_to_json(array_agg(f)) AS features
+          FROM(
+          SELECT 
```

Comment:
there should be an extra indentation here (and a space after the "FROM")

Reply:
Also the same applies for all persistence queries

Reply:
Fixed